Dockerfile

```diff
@@ -117,6 +117,9 @@ RUN CHART_VERSION="v1.18.10"     CHART_FILE=/charts/rke2-kube-proxy.yaml
 RUN CHART_VERSION="2.11.100"    CHART_FILE=/charts/rke2-metrics-server.yaml    CHART_BOOTSTRAP=false   /charts/build-chart.sh
 RUN rm -vf /charts/*.sh /charts/*.md
 
+FROM manifests AS manifests
+COPY manifests/ccm.yaml /manifests/
```

Comment:
We are not packaging roles and such as manifest files for rke2. The only thing we deploy should be helm charts. If you need to add RBAC it should be added in here somewhere: https://github.com/rancher/rke2/blob/master/pkg/rke2/clusterrole_templates.go